schemas/common/accesscontrolpolicy.example.1.json

```diff
@@ -0,0 +1,14 @@
+ {
+  "acl": [
```

Comment:
These look like they will need prefixes.

Reply:
Yes, we have three kinds of properties:

1. core XDM properties (they have a `:` somewhere)
2. extension properties (they look like URLs)
3. special sauce properties (`@id`, `@type`, and maybe `_links` and `_embedded`)

Let's say, I like the consistency of it.

Reply:
ack